deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -51,38 +51,37 @@ serviceMonitors:
 
     ## Endpoints of the selected service to be monitored
     ## endpoints: []
-       ## Name of the endpoint's service port
-       ## Mutually exclusive with targetPort
-       # - port: ""
+    ## Name of the endpoint's service port
+    ## Mutually exclusive with targetPort
+    #    - port: ""
 
-       ## HTTP path to scrape for metrics
-       ##
-       #   path: /metrics
+    ## HTTP path to scrape for metrics
+    ##
+    #      path: /metrics
     endpoints:
       - port: exporter
         path: /mertics
 
     ## Label selector for services to which this ServiceMonitor applies
     ## selector: {}
-       # matchLabels: {}
-       # Eg- matchLabels:
-       #       cas-type: cstor
+    #    Eg- matchLabels:
+    #    matchLabels: {}
+    #      cas-type: cstor
     selector:
       matchLabels:
         openebs.io/cas-type: cstor
 
     ## Namespaces from which services are selected
     ## namespaceSelector: []
-       ## Match any namespace
-       ##
-       # any: true
-
-       ## Explicit list of namespace names to select
-       ##
-       # matchNames: []
-       ## Eg- matchNames:
-       #     - openebs
-       #     - default
+    ## Match any namespace
+    #    any: true
+
+    ## Explicit list of namespace names to select
+    ##
+    #    matchNames: []
+    ## Eg- matchNames:
+    #        - openebs
+    #        - default
     namespaceSelector:
       any: true
 
```

Comment:
Please change the below jiva enabled field to `true` to enable jiva monitoring by default.

Reply:
Done.